INSTALL_MACOS.md

```diff
@@ -96,159 +107,189 @@ EOF
+Patching a Homebrew recipe is explained in the [Homebrew FAQ](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/FAQ.md).
 
-    brew edit cartr/qt4/qt@4
+```shell
+brew edit cartr/qt4/qt@4
+```
 
 and before the line that starts with `head`, add the following code:
 
-      patch :p0 do
-        url "https://bugreports.qt.io/secure/attachment/52520/patch-qthread-stacksize.diff"
-        sha256 "477630235eb5ee34ed04e33f156625d1724da290e7a66b745611fb49d17f1347"
-      end
-
+```
+patch :p0 do
+  url "https://bugreports.qt.io/secure/attachment/52520/patch-qthread-stacksize.diff"
+  sha256 "477630235eb5ee34ed04e33f156625d1724da290e7a66b745611fb49d17f1347"
+end
+```
 
 Install libraries:
 
-    brew install cairo expat
-    brew install gnu-sed gawk coreutils findutils
-    brew install cmake keychain sphinx-doc
-    /usr/local/opt/sphinx-doc/libexec/bin/pip3 install sphinx_rtd_theme
-    brew install --build-from-source qt --with-mysql
+```shell
+brew install cairo expat
+brew install gnu-sed gawk coreutils findutils
+brew install cmake keychain sphinx-doc
+/usr/local/opt/sphinx-doc/libexec/bin/pip3 install sphinx_rtd_theme
+brew install --build-from-source qt --with-mysql
+```
 
 On macOS Sierra, install the sierra-compatible recipe (to be used only in Sierra, since this builds Qt from sources and takes a while):
 
-    brew install --build-from-source cartr/qt4/qt --with-mysql
+```shell
+brew install --build-from-source cartr/qt4/qt --with-mysql
+```
 
-Then install pyside (the boneyard tap is for pyside, which does not yet build with Qt5 and was thus removed from the homebrew core):
+Then install pyside (the boneyard tap is for pyside, which does not yet build with Qt5 and was thus removed from the Homebrew core):
 
-    brew install pyside@1.2 pyside-tools@1.2
+```shell
+brew install pyside@1.2 pyside-tools@1.2
+```
 
 The last command above will take a while, since it builds from sources, and should finally tell you do do the following if the `homebrew.pth` file does not exist:
 
-    mkdir -p ~/Library/Python/2.7/lib/python/site-packages
-    echo 'import site; site.addsitedir("/usr/local/lib/python2.7/site-packages")' >> ~/Library/Python/2.7/lib/python/site-packages/homebrew.pth
-    sudo ln -s ~/Library/Python/2.7/lib/python/site-packages/homebrew.pth /Library/Python/2.7/lib/python/site-packages/homebrew.pth
+```shell
+mkdir -p ~/Library/Python/2.7/lib/python/site-packages
+echo 'import site; site.addsitedir("/usr/local/lib/python2.7/site-packages")' >> ~/Library/Python/2.7/lib/python/site-packages/homebrew.pth
+sudo ln -s ~/Library/Python/2.7/lib/python/site-packages/homebrew.pth /Library/Python/2.7/lib/python/site-packages/homebrew.pth
+```
 
  To install the [openfx-io](https://github.com/NatronGitHub/openfx-io) and [openfx-misc](https://github.com/NatronGitHub/openfx-misc) sets of plugin, you also need the following:
 
-    brew install ilmbase openexr freetype fontconfig ffmpeg opencolorio openimageio
```

Comment:
Natron won't compile with the most recent version of seexpr, why was this part removed?

Reply:
This was an error, will fix.